Review base stability

mnsnor opened the pull request.

mnsnor committed:
Create BASE_stability.tex

The stability analysis on the movable base

mnsnor committed:
Add files via upload

mnsnor committed:
Update BASE_stability.tex

Forgot to put in the discrete images

edwced commented on report/finished/chapters/BASE_stability.tex:
```diff
@@ -0,0 +1,60 @@
+    G_{cl}(s)=\frac{G_{ol}(s)}{1+G_{ol}(s)}=\frac{(2r_b s)}{(2r_b s)*(2r_b s + 1)}
+\end{equation}
+
+\noindent The function has a pole and zero cancellation at 0. In Fig.\ref{fig:step1} the step response, poles and zeros are plotted for the closed loop system. Since all the poles appeare on the negative real plane, the system is stable.
```
= zero

edwced replied:
we might want to have an reference on why this makes it stable

NiklasDahlquist1 replied:
what function? Reference it.

edwced commented on report/finished/chapters/BASE_stability.tex:
```diff
@@ -0,0 +1,60 @@
+\end{equation}
+
+\noindent From this it's easy to see that the system will remain stables for $k_d > - 2 r_b$ and that the value of $k_d$ won't have any effect on the stability of the system in continuous time.\\
+\indent The values for $k_p$ and $k_d$ is therefore chosen in such a manner that $ 0 > k_d > -2 r_b$ and $k_p$ so that the greatest error input from the \textbf{MV}-node will not saturate the actuators. The result of these are shown in Fig.\ref{fig:step2} and a comparison was made in table \ref{tab:with_without}
```
was made, represented in table

edwced commented on report/finished/chapters/BASE_stability.tex:
```diff
@@ -0,0 +1,60 @@
+\begin{figure}[H]
+    \centering
+    \includegraphics[width= 0.7\textwidth]{img/closed_loop_step_PD.eps}
+    \caption{The closed loop system with a controller}
```
. end of caption, if we do that.

edwced reviewed (commented):
nice !

NiklasDahlquist1 commented on report/finished/chapters/BASE_stability.tex:
```diff
@@ -0,0 +1,67 @@
+
+\begin{equation}
+    G_{cl}(s)=\frac{G_{ol}(s)}{1+G_{ol}(s)}=\frac{(2r_b s)}{(2r_b s)*(2r_b s + 1)}
+\end{equation}
```
End sentence with a period (.)

NiklasDahlquist1 commented on report/finished/chapters/BASE_stability.tex:
```diff
@@ -0,0 +1,67 @@
+\noindent The \textbf{PD}-controller discussed in previous sections only need to improve the system response, since the overall system is already stable. Introducing the controller to the system gives us the closed loop function:
+
+\begin{equation}
+    G_{clPD} = \frac{2r_b s(k_p + k_d s)}{2r_b s(s(2r_b+k_d)+k_p)}
```
End sentence with a period (.)

NiklasDahlquist1 commented on report/finished/chapters/BASE_stability.tex:
```diff
@@ -0,0 +1,67 @@
+        Settling Time & 1.5648$s$ & 0.1565$s$ & 0.2500$s$ \\
+        \hline
+    \end{tabular}
+    \caption{Comparison between the closed loop system}
```
I think the caption should be at the top of a table?

NiklasDahlquist1 commented on report/finished/chapters/BASE_stability.tex:
```diff
@@ -0,0 +1,67 @@
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width= 0.7\textwidth]{img/closed_loop_step_D.eps}
```
Add legends to the plot.

NiklasDahlquist1 commented on report/finished/chapters/BASE_stability.tex:
```diff
@@ -0,0 +1,67 @@
+\begin{figure}[H]
+    \centering
+    \includegraphics[width= 0.7\textwidth]{img/closed_loop_step_D.eps}
+    \caption{The discrete step response and the poles and zeros of the discrete transfer function}
```
Add that the continuous response is included in the figure.

NiklasDahlquist1 reviewed (commented):
Add periods to the captions

mnsnor committed:
Update BASE_stability.tex

Fixed everything except a reference for the stability of the discrete time case. Will look for a source to back that up later tonight

mnsnor committed:
Add files via upload

mnsnor committed:
Update BASE_stability.tex

mnsnor committed:
Update bibliography.bib

mnsnor committed:
Update BASE_stability.tex

Added citation for the discrete time stability check
